### PR TITLE
Add docs about new compression explorer in Cloud Console

### PR DESCRIPTION
**Pull request by charislam**
# Description

Add docs about new compression explorer in Cloud Console

# Links

Fixes https://github.com/timescale/docs-private/issues/58

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: cloud/service-explorer.md [hunk 1]
```diff
@@ -45,13 +45,48 @@ aggregates, and policies such as data retention policies and data reordering.
 You can also inspect individual hypertables, including their sizes, dimension
 ranges, and compression status.
 
+You can also set a compression policy from this section. See [Setting a
+compression policy from Timescale Cloud console][set-compression] for details.
```
Comment (Loquacity) [inline cloud/service-explorer.md:49]:
```suggestion
You can also set a compression policy from this section. For more information, see the 
[Setting a compression policy from Timescale Cloud console][set-compression] section.
```

File: cloud/service-explorer.md [hunk 2]
```diff
@@ -45,13 +45,48 @@ aggregates, and policies such as data retention policies and data reordering.
...
+1.  In the `Compression policy` section, click `Create compression policy`. A
+    suggested policy interval is pre-selected for you, but you can change this
+    in the next step.
+
+    <img class="main-content__illustration"
+    src="https://s3.amazonaws.com/assets.timescale.com/docs/images/tsc-explorer-compression-policy.png"
+    alt='A screenshot of the compression policy section, showing the "Create
+    compression policy" button and a default policy interval of 14 days.' />
+
+1.  A modal appears with options for different compression policy intervals.
```
Comment (Loquacity) [inline cloud/service-explorer.md:78]: 🪄

**Review by brandon-sprelo — COMMENTED** [review]
looks good

**Comment by charislam** [comment]
@Brandon-Timescale this is behind a feature flag right? When should we publish the docs?

**Comment by Brandon-Timescale** [comment]
> @Brandon-Timescale this is behind a feature flag right? When should we publish the docs?

We're hoping to remove the FF tomorrow or Wednesday. It should be fine to publish now then release?

**Review by Loquacity — APPROVED** [review]

**Pull request merged**